aperturedb/Connector.py

```diff
@@ -113,7 +114,8 @@ def __init__(self, host="localhost", port=55555,
                  use_keepalive=True,
                  retry_interval_seconds=1,
                  retry_max_attempts=3,
-                 config: Optional[Configuration] = None):
+                 config: Optional[Configuration] = None,
+                 encoded: Optional[str] = None):
```

Comment:
Can we just call this token?

Reply:
we absolutely can. I think I called it "encoded" just because "token" is already overloaded in our codebase with the token generation for account access, etc - 

if we make sure people know this is an "connect token" or something hopefully it wouldn't cause issues.